src/analytics/ValoraAnalytics.ts

```diff
@@ -77,9 +86,8 @@ class ValoraAnalytics {
       try {
         const deviceInfo = await getDeviceInfo()
         this.deviceInfo = deviceInfo
-        this.sessionId = sha256FromString(
-          '0x' + deviceInfo.UniqueID.split('-').join('') + String(Date.now())
-        )
+        uniqueID = await deviceInfo.UniqueID
```

Comment:
you don't need to await this. it's not a promise.

Reply:
```suggestion
        uniqueID = deviceInfo.UniqueID
```

Reply:
This is a promise in the new version of `react-native-device-info`. Actually, I'll just change [`getDeviceInfo`](https://github.com/valora-inc/wallet/blob/main/src/analytics/ValoraAnalytics.ts#L41) so that the await happens in there.